src/SFA.DAS.EAS.Portal/Database/Models/Account.cs

```diff
@@ -0,0 +1,116 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using Newtonsoft.Json;
+using SFA.DAS.EAS.Portal.Client.Models;
+
+namespace SFA.DAS.EAS.Portal.Database.Models
+{
+    public class Account : Document, IAccountDto
+    {
+        [JsonProperty("accountId")]
+        public long AccountId { get; private set; }
+
+        [JsonProperty("accountLegalEntities")]
+        public IEnumerable<IAccountLegalEntityDto> AccountLegalEntities => _accountLegalEntities;
+
+        [JsonProperty("outboxData")]
+        public IEnumerable<OutboxMessage> OutboxData => _outboxData;
+        
+        [JsonProperty("created")]
+        public DateTime Created { get; private set; }
+
+        [JsonProperty("updated")]
+        public DateTime? Updated { get; private set; }
+
+        [JsonProperty("deleted")]
+        public DateTime? Deleted { get; private set; }
+
+        [JsonIgnore]
+        private readonly List<AccountLegalEntity> _accountLegalEntities = new List<AccountLegalEntity>();
+
+        [JsonIgnore]
+        private List<OutboxMessage> _outboxData = new List<OutboxMessage>();
+
+        private Account(long accountId, DateTime created, string messageId)
+            : base(1)
+        {
+            Id = Guid.NewGuid();
+            AccountId = accountId;
+            // the created date originates from event publishers, so we make sure we store it as UTC
+            Created = DateTime.SpecifyKind(created, DateTimeKind.Utc);
+            
+            AddOutboxMessage(messageId, created);
+        }
+
+        // ctor for when creating account doc from reserved funding
+        // do we accept event, seems a coupling too far!
+        public Account(long accountId, long accountLegalEntityId, string legalEntityName, Guid reservationId,
```

Comment:
Looks very RPC like.  Doesn't support easy changes. Better to pass in objects.  Or leave it to the handlers to update the read model based on the events received. Add as tech debt in 454.